setup.py

```diff
@@ -14,6 +14,8 @@
     'cloudpickle',
     # Deprecation management
     'deprecation>=2',
+    # For status view markdown rendering
+    'mistune'
```

Comment:
Is there a minimum version we need for this to work?

Also, I would suggest making this a soft requirement by using [`extras_require`](https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies) instead.

Reply:
I modified it using extras_require following what I found online, but not entirely sure, would be great if you take another look on this.